test/tests/time_integrators/central-difference/ad_central_difference_dotdot.i

```diff
@@ -0,0 +1,86 @@
+###########################################################
+# This is a simple test with a time-dependent problem
+# demonstrating the use of the TimeIntegrator system.
+#
+# Testing that the second time derivative is calculated
+# correctly using the Central Difference method for an AD
+# variable.
+#
+# @Requirement F1.30
```

Comment:
This is fancy. What does this mean?

Reply:
I am not entirely sure....precursor to the new SQA requirements I think, since this is copypasta from an old test. I will remove in the same commit that I switch to `inline` functions for `adUDotDot`

Reply:
Resolved in 2fdf84b